### ISSUE TITLE
Randomly dying for no reason

### ISSUE THREAD

**Issue by ghost** [Terasology/JoshariasSurvival#33]
I was hopping up a mountain when all the sudden I died from "physical damage". I later died when walking through one block deep water along a beach. It seems to be random.

**Comment by skaldarnar** [comment]
Thanks for the report @AmeliaMayIX. 

I think @misterHippo observed similar death-experiences, while I was not able to reproduce it (at all). Debugging this issue is really hard if we cannot find a way to reprodcue 🤔 

This might be related to movement, movement speed, and fall/collision damage. 

Could you post the specs of your computer here, as long with a note on how smoothly the game runs (e.g., average fps), please?

**Comment by ghost** [comment]
The game always runs smoothly for me, though I don't have data on the actual FPS.

Specs:
OS Name: Microsoft Windows 10 Home
System Type: x64-based PC
Processor: AMD Ryzen 7 1800X Eight-Core Processor, 3600 Mhz, 8 Cores, 16 Logical Processors
RAM: 16.0 GB

**Comment by misterHippo** [comment]
Yeah, we did have a similar thing. The kids and I now have a joke where we bump into each other, or brush against a wall... or like, turn to face someone else; and then we say "[someone] died of physical damage". #trueStory

My setup is headless server on CentOS 7 machine (mobile intel processor, 4GB ram), client is 16GB Ram, core i5-4690 (3.5GHz). Server reboots nightly - as lengthy run times caused issues. Don't recall FPS at the time. (Oh! also version was nightly 34.)

@skaldarnar - Did I open an issue about this? I can't remember / find it.
